data-serving/data-service/api/dataserver.api.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.3
+info:
+  title: EPID data service
+  description: Server for CRUD operations on EPID line-list data
+  termsOfService: 'https://www.healthmap.org/about/termsofuse/'
+  contact:
+    email: info@healthmap.org
+  license:
+    name: MIT
+  version: 1.0.0
```

Comment:
Worth adding a `servers` object pointing to the port we've designated on localhost? Seems reasonably canonical to do so, and slap a `description: Local server` on it.

Reply:
good call, thanks!